testing/dart/compositing_test.dart

```diff
@@ -28,7 +28,7 @@ void main() {
     assert(() {
       expect(
         () => builder.pushTransform(matrix4WrongLength),
-        throwsA(const TypeMatcher<AssertionError>()),
+        throwsA(isInstanceOf<AssertionError>()),
```

Comment:
These are all candidates for simplification to `throwsAssertionError`, which we test for frequently.

Reply:
Done.